Add stc

alexonab opened the pull request.

peerchemist commented:
Why did you make so many changes in the finta.py? It looks like a lot of changed code formatting.

alexonab commented:
What happened was after I added the indicator and the test, I ran black on finta.py and committed the change which caused the formatting changes.  Once I noticed the formatting changes I reverted to the original without the changes made by black.

peerchemist commented:
Can you please make a rebase?
